extensions/amp-story-shopping/0.1/test/test-amp-story-shopping.js

```diff
@@ -0,0 +1,38 @@
+import {Services} from '#service';
+import {LocalizationService} from '#service/localization';
+
+import {formatI18nNumber} from '../amp-story-shopping';
+
+describes.realWin(
+  'amp-story-shopping',
+  {
+    amp: {runtimeOn: true},
+  },
+  (env) => {
+    let win;
+    let element;
+    let localizationService;
+
+    beforeEach(async () => {
+      win = env.win;
+
+      localizationService = new LocalizationService(win.document.body);
+      env.sandbox
+        .stub(Services, 'localizationServiceForOrNull')
+        .returns(Promise.resolve(localizationService));
+
+      element = win.document.createElement('amp-story-page');
+      element.setAttribute('lang', 'en');
+    });
+
+    it('should format i18n number', () => {
```

Comment:
Since this method is meant to test i18n values, it could be worthwhile to add a second currency to this test, or at least a non-USD currency